admin/includes/functions/general.php

```diff
@@ -909,7 +909,7 @@ function zen_set_product_status($products_id, $status) {
 // Sets timeout for the current script.
 // Cant be used in safe mode.
   function zen_set_time_limit($limit) {
-    if (version_compare(PHP_VERSION, 5.4, '>=') || !get_cfg_var('safe_mode')) {
+    if (!get_cfg_var('safe_mode')) {
```

Comment:
do we even need the `safe_mode` check anymore?

Reply:
I do see it once in a while.  (Of course I tell the client to switch, but I do see it.)

Reply:
But, in the sqlpatch update you removed it. 
Let's be consistent and remove it in both places. It was deprecated in PHP 5.3 and removed in PHP 5.4

Reply:
LOL you're right.  So much for keeping track of time.  OK removed.